dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.dubbo.admin.common.util;
```

Comment:
Please add Apache License announcement at the head of new files.
You can copy it from any other sourcecode.

Reply:
ok